src/main/java/goral/psychotherapistoffice/web/admin/TherapyManagementController.java

```diff
@@ -27,7 +27,7 @@ public String addTherapyForm(Model model) {
         TherapyDto therapyDto = new TherapyDto();
         model.addAttribute("therapyDto", therapyDto);
         model.addAttribute("heading",
-                "Dodaj rodzaj terapii, opis, cenę. Nie wpisuj Id, zostanie wygenerowane autoamtycznie");
+                "Dodaj rodzaj terapii, opis, cenę. Nie wpisuj Id, zostanie wygenerowane automatycznie.");
```

Comment:
"Nie wpisuj Id" maybe should we hide id input?

Reply:
I don't wont create new template for edit add ad therapy so I add only this text in controller and use the same templete;) 
I will fix it:)

Reply:
we can use same template and add if else condition